automation/taskcluster/decision_task.py

```diff
@@ -126,7 +115,7 @@ def create_compare_locales_task():
 
     queue = taskcluster.Queue({ 'baseUrl': 'http://taskcluster/queue/v1' })
```

Comment:
`baseUrl` is deprecated in taskcluster-client.py for 4.*, and removed in 5.* (https://github.com/taskcluster/taskcluster-client.py/blob/master/taskcluster/client.py#L58).  Maybe this is a good time to change to `rootUrl`, the new option. I don't know where the module version is specified for this deployment.

Reply:
IMHO, I'd like to keep this PR specific to updating how we source artifact metadata.
Is there a good place to note that this needs to be updated? @MihaiTabara mentioned a trello this morning, or is bugzilla a better approach? @JohanLorenzo too :)

Reply:
Agreed on keeping this PR specific :+1: 

https://bugzilla.mozilla.org/show_bug.cgi?id=1460015 tracks this issue. IMO, this is good enough for now. We can revisit this, if this becomes too much of a hassle.

Reply:
I've added an item in trello as well to keep this under the radar.  https://trello.com/c/aWjXAAke/10-once-the-taskcluster-github-integration-is-solved-with-taskcluster-5-we-should-update-it